auslib/web/admin/views/base.py

```diff
@@ -25,34 +23,39 @@ def decorated(*args, **kwargs):
             try:
                 return f(*args, **kwargs)
             except OutdatedDataError as e:
-                msg = "Couldn't perform the request %s. Outdated Data Version. old_data_version doesn't match current data_version" % messages
+                msg = "Couldn't perform the request %s. Outdated Data Version. " \
+                      "old_data_version doesn't match current data_version" % messages
                 logging.warning("Bad input: %s", msg)
                 logging.warning(e)
-                return Response(status=400, response=json.dumps({"exception": msg}), mimetype="application/json")
+                # using connexion.problem results in TypeError: 'ConnexionResponse' object is not callable
+                # hence using flask.Response but modifying response's json data into connexion.problem format
+                # for validation purpose
```

Comment:
This comment could be improved. It talks about "using flask.Response", but there's no references to it here. Maybe this should be documented in problem.py instead?